molecule/opensearch/converge.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: Converge
+  hosts: all
+  tasks:
+    - name: "Include alfresco-ansible-deployment"
+      include_role:
```

Comment:
🤔 should'nt execute the entire playbook?

Reply:
leftovers from the molecule init